examples/ic_io/retimer_PI2EQX22024_output.json

```diff
@@ -1,1786 +1,1900 @@
+              ],
+              "maxValue": 1800,
+              "siUnit": "microamp",
+              "typValue": 800
             },
-            "absVmin": {
-                "values": [
-                    {
-                        "siUnit": "volt",
-                        "minValue": -0.3
-                    }
-                ]
+            {
+              "conditions": [
+                "EN=1",
+                "USB U2/U3 mode",
+                "VDD= 1.8V",
+                "2 Lane"
+              ],
+              "maxValue": 2000,
+              "siUnit": "microamp",
+              "typValue": 900
             },
-            "externalComponents": [
-                {
-                    "componentType": "capacitor",
-                    "configuration": "pd",
-                    "values": [
-                        {
-                            "conditions": [
-                                "External VDD1V_CON1/2 decoupling capacitors with max ±20% tolerance"
-                            ],
-                            "siUnit": "nanofarad",
-                            "maxValue": 15
-                        }
-                    ]
-                }
-            ]
+            {
+              "conditions": [
+                "EN=1",
+                "USB RXDET mode",
+                "VDD =1.8V",
+                "1 Lane"
+              ],
+              "maxValue": 1700,
+              "siUnit": "microamp",
+              "typValue": 735
+            },
+            {
+              "conditions": [
+                "EN=1",
+                "USB RXDET mode",
+                "VDD =1.8V",
+                "2 Lane"
+              ],
+              "maxValue": 1900,
+              "siUnit": "microamp",
+              "typValue": 800
+            }
+          ]
         },
+        "supplyName": "VDD1P8"
+      }
+    ],
+    "integratedAuxLsSwitch": false,
+    "interface": [
+      "USB3.2 Gen2x1",
+      "USB3.2 Gen 2x2"
+    ],
+    "maxDataRate": {
+      "values": [
         {
-            "terminalIdentifier": "4",
-            "name": "AP_TX2P",
-            "description": "TX CML output terminals.Output termination 50/1.5kΩ to VbiasTX, 3kΩ to GND or 75kΩ to GND",
-            "functionProperties": [
-                {
-                    "direction": "out"
-                }
-            ],
-            "absVmax": {
-                "values": [
-                    {
-                        "siUnit": "volt",
-                        "maxValue": 1.15
-                    }
-                ]
+          "maxValue": 10,
+          "unitName": "Gbps"
+        }
+      ]
+    },
+    "numberOfLanes": 2,
+    "partType": "retimer",
+    "sris": false,
+    "srns": false
+  },
+  "pins": [
+    {
+      "terminalIdentifier": [
+        "1"
+      ],
+      "name": "AP_RX2P",
+      "description": "RX CML input terminals.Input with termination 50Ω to GND or 75kΩ to GND",
+      "functionProperties": [
+        {
```

Comment:
functionProperties no longer required here and "direction" can be moved within pinProperties.